### ISSUE TITLE
Ray 2.5.0 broke command-line support on Windows

### ISSUE THREAD

**Issue by vnlitvinov** [conda-forge/ray-packages-feedstock#100]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

After installing `ray-default==2.5.0` from conda-forge, I cannot execute `ray` command, as it fails with a cryptic error message
```
Fatal error in launcher: Unable to create process using '"D:\bld\ray-packages_1687348866507\_h_env\python.exe"  "C:\mambaforge\envs\modin\Scripts\ray.exe"
```
(here `C:\mambaforge\envs\modin` is my conda environment, and I believe `D:\bld` corresponds to where it was built in the CI)

cc @krfricke @h-vetinari @aslonnie

### Installed packages

```shell
ray-default               2.5.0           py310h5588dad_0    conda-forge
```

### Environment info

```shell
active environment : modin
    active env location : C:\mambaforge\envs\modin
            shell level : 2
       user config file : C:\Users\Vass\.condarc
 populated config files : C:\mambaforge\.condarc
          conda version : 22.9.0
    conda-build version : not installed
         python version : 3.10.6.final.0
       virtual packages : __win=0=0
                          __archspec=1=x86_64
       base environment : C:\mambaforge  (writable)
      conda av data dir : C:\mambaforge\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\mambaforge\pkgs
                          C:\Users\Vass\.conda\pkgs
                          C:\Users\Vass\AppData\Local\conda\conda\pkgs
       envs directories : C:\mambaforge\envs
                          C:\Users\Vass\.conda\envs
                          C:\Users\Vass\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/22.9.0 requests/2.28.1 CPython/3.10.6 Windows/10 Windows/10.0.22621
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by vnlitvinov** [comment]
ref: https://github.com/ray-project/ray/pull/36547

**Comment by vnlitvinov** [comment]
Wild guess - might be caused by https://github.com/conda-forge/ray-packages-feedstock/commit/d5fdaf4687402e9d29350470ed64193fcc728e74 (as this is the only meaningful change on how we build 2.4 vs 2.5, and on 2.4 it certainly works fine).

This issue also means we'd have to add a test running something like `ray --help` to this recipe.

**Comment by aslonnie** [comment]
fwiw, https://github.com/ray-project/ray/pull/36547 is not merged yet.

**Comment by vnlitvinov** [comment]
Fixed via #102.
